### PR TITLE
Be sure to write cnd urls for both instances

### PR DESCRIPTION
**Pull request by D4nte**
Doh!

### CODE REVIEW

File: src/docker/cnd.rs [hunk 1]
```diff
@@ -1,6 +1,6 @@
 use crate::docker::{ExposedPorts, Image};
 
-const HTTP_URL_CND: &str = "HTTP_URL_CND";
+const HTTP_URL_PREFIX: &str = "HTTP_URL_";
```
Comment (luckysori) [inline src/docker/cnd.rs]: :question: 
```suggestion
const HTTP_URL_PREFIX: &str = "HTTP_URL_CND_";
```

File: src/docker/cnd.rs [hunk 2]
```diff
@@ -12,11 +12,15 @@ impl Image for Cnd {
         vec!["--", "cnd", "--config=/config/cnd.toml"]
     }
 
-    fn expose_ports() -> Vec<ExposedPorts> {
+    fn expose_ports(name: &str) -> Vec<ExposedPorts> {
```
Comment (luckysori) [inline src/docker/cnd.rs]: :question: Name? Isn't just an index?

Comment (D4nte) [reply src/docker/cnd.rs]: no it's a name... :)

Comment (D4nte) [reply src/docker/cnd.rs]: same name than the container.

File: src/docker/cnd.rs (hunk 1 above)
Comment (D4nte) [reply src/docker/cnd.rs]: Nope

**Comment by D4nte** [comment]
Confirmed working against hello-swap master!

**Pull request merged**